app/templates/partials/applications_table.html

```diff
@@ -0,0 +1,101 @@
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{%- from "govuk_frontend_jinja/components/tag/macro.html" import govukTag -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+
+{% macro applications_table(applications, is_past_submission_deadline) %}
+
+{% set ns = namespace(rows = []) %}
+    {% for application in applications %}
+    {% set status_class %}
+    {% if is_past_submission_deadline %}
+        {% if application.status == 'SUBMITTED' %}
+            complete-tag
+        {% elif application.status == 'NOT_SUBMITTED' %}
+            complete-tag
+        {% endif %}
+    {% endif %}
+    {% if not is_past_submission_deadline %}
```

Comment:
Could probably just be an else:

```suggestion
    {% else %}
```

Reply:
Just realized this in the copied file...